opennlp-tools/src/main/java/opennlp/tools/tokenize/lang/Factory.java

```diff
@@ -43,18 +53,25 @@ public class Factory {
    */
   public Pattern getAlphanumeric(String languageCode) {
```

Comment:
At some point would it make sense to have `languageCode` be an enum instead of a string?

Reply:
Feel free to open a Jira _epic_ (or single issue?) for this improvement and share a list of languages most widely used by OpenNLP's users. As a start, this `Factory` might cover most.

Reply:
A good idea I think, +1 to both the enum and the jira (probably an epic, especially if we need to update sandbox components).